CHANGELOG.rst

```diff
@@ -51,6 +57,7 @@ CHANGELOG
 * Remove unused css from delete confirmation view and move file view
 * Add Pillow 10 compatibility
 * Update translations (de/fr/nl)
+* Drop Django 2.2 support
```

Comment:
```suggestion
* Drop Django 2.2, 3.0, and 3.1 support
```

Reply:
Done in f5532518775c8c99dbced814676fe834824ae967 :)